framework/src/multiapps/MultiApp.C

```diff
@@ -120,6 +120,14 @@ MultiApp::validParams()
                                 1,
                                 "Minimum number of processors to give to each App in this "
                                 "MultiApp.  Useful for larger, distributed mesh solves.");
+  params.addParam<bool>(
+      "quiet_app_init",
```

Comment:
it s more "staggered" than quiet?

but even staggered would be 1 by 1 not 1 then n-1
suggestions? wish there was a "volunteer-naming-suggestions" group we could tag haha

Reply:
Yeah I was not sure of a good name here - "staggered" is perhaps closer to the intention... `barrier_until_first_app_initialized` is probably the most literal choice?

Reply:
`wait_for_first_app`

Reply:
works
wait_for_first_app_init ?

Reply:
Either one is fine with me. I think they're both a little more intuitive

Reply:
I went with `wait_for_first_app_init`, I agree those are better names.